### PR TITLE
Clarify timestamping descriptions [AP-1087]

### PR DESCRIPTION
**Pull request by reimerix**
# Description

@swift-nav/devinfra

# API compatibility

Does this change introduce a API compatibility risk?

## API compatibility plan

If the above is "Yes", please detail the compatibility (or migration) plan:

# JIRA Reference

https://swift-nav.atlassian.net/browse/AP-1087

### CODE REVIEW

**Review by pcrumley — APPROVED** [review]
Thanks for catching these grammatical issues!

**Review by pcrumley — COMMENTED** [review]
we'll need to bump the build-libsbp docker tag to get this to pass ci. I'm not sure the best way to get in changes which change the generated code (tagging @woodfell because he changed the generated ci checks recently)

**Comment by woodfell** [comment]
No, I don't think this is a tag issue. Not all languages have been generated. This PR is (I think) missing:
- java 
- jsonschema
- protobuf
- python
- quicktype

File: c/include/libsbp/legacy/system.h [hunk 1]
```diff
@@ -217,15 +217,16 @@ typedef struct SBP_ATTR_PACKED SBP_DEPRECATED {
...
  * (e.g. IMU or wheeltick messages) where GNSS time is unknown to the sender.
  *
  * The local time used to timestamp the PPS pulse must be generated by the
  * same clock which is used to timestamp the IMU/wheel sensor data and should
- * follow the same roll-over rules.  A separate MSG_PPS_TIME message should be
- * sent for each source of sensor data which uses PPS-relative timestamping.
- * The sender ID for each of these MSG_PPS_TIME messages should match the
- * sender ID of the respective sensor data.
+ * follow the same roll-over rules (i.e. it should roll over to zero  after
+ * 604800 seconds). A separate MSG_PPS_TIME message should be sent  for each
```
Comment (dgburr) [inline c/include/libsbp/legacy/system.h:226]: Extra space

File: spec/yaml/swiftnav/sbp/system.yaml [hunk 2]
```diff
@@ -451,18 +451,18 @@ definitions:
...
 
       The local time used to timestamp the PPS pulse must be generated by the
       same clock which is used to timestamp the IMU/wheel sensor data and
-      should follow the same roll-over rules.  A separate MSG_PPS_TIME
-      message should be sent for each source of sensor data which uses
-      PPS-relative timestamping.  The sender ID for each of these
-      MSG_PPS_TIME messages should match the sender ID of the respective
-      sensor data.
+      should follow the same roll-over rules (i.e. it should roll over to zero 
+      after 604800 seconds). A separate MSG_PPS_TIME message should be sent
```
Comment (dgburr) [inline spec/yaml/swiftnav/sbp/system.yaml:462]: @reimerix: Space character at the end of this line (and the one above) should be removed

**Review by woodfell — APPROVED** [review]

**Pull request merged**